dist/styles/table.scss

```diff
@@ -1,13 +1,17 @@
+.mdc-data-table {
+  width: 100%;
+}
+
 table {
   border: {
     spacing: 0;
   }
   max: {
     width: 100%;
   }
-  table: {
-    layout: fixed;
-  }
+  // table: {
+    // layout: fixed;
+  // }
```

Comment:
uncomment and adjust the widths to the below

```
&.width-50 {
          width: 38%;
        }

        &.width-5 {
          width: 7%;
        }

        &.width-9 {
          width: 11%;
        }
```

Reply:
See my argument above about forcing fixed concepts about browser environments above but I'll make this change.  😄

Reply:
👍